src/main/java/hudson/remoting/jnlp/Main.java

```diff
@@ -92,6 +89,10 @@
             usage="If the connection ends, don't retry and just exit.")
     public boolean noReconnect = false;
 
+    @Option(name="-insecure",
+        usage="Ignore SSL validation errors - use as a last resort only.")
```

Comment:
Exactly the type of comment I would like to see more often on such option :) :+1: